data/brands/amenity/charging_station.json

```diff
@@ -200,6 +200,30 @@
         "operator": "Recharge"
       }
     },
+    {
+      "displayName": "Rivian Adventure Chargers",
+      "id": "rivian-dd8b2e",
+      "locationSet": {"include": ["us"]},
+      "tags": {
+        "amenity": "charging_station",
+        "brand": "Rivian",
+        "brand:wikidata": "Q7338847",
+        "network": "Rivian",
+        "network:wikidata": "Q7338847"
+      }
+    },
+    {
+      "displayName": "Rivian Waypoints",
+      "id": "rivianwaypoints-dd8b2e",
```

Comment:
```
🏗   Loading index files...
Error - Duplicate id 'rivian-5598dd' in:
  Rivian Waypoints
  data/brands/amenity/charging_station.json
ERROR: "build:index" exited with 1.
ERROR: "build" exited with 1.
Error: Process completed with exit code 1.
```
This id will be overwritten while building index by anothe `rivian-5598dd` as the same as above entry, and this is not allowed. That's because they have the same `brand` value, and this entry is in `/data/brands/..`

Reply:
Thank you for telling me this, as I did not know how or why the errors occurred.